pkg/distro/rhel9/edge.go

```diff
@@ -196,7 +198,19 @@ var (
 		// TODO(runcom): move fdo-client-linuxapp.service to presets?
 		"NetworkManager.service", "firewalld.service", "sshd.service", "fdo-client-linuxapp.service",
 	}
-
+	//dropin to disable grub-boot-success.timer if greenboot present
+	systemdUnits = []*osbuild.SystemdUnitStageOptions{
+		{
+			Unit:     "grub-boot-success.timer",
+			Dropin:   "10-disable-if-greenboot.conf",
+			UnitType: osbuild.Global,
+			Config: osbuild.SystemdServiceUnitDropin{
+				Unit: &osbuild.SystemdUnitSection{
+					FileExists: "!/usr/libexec/greenboot/greenboot",
+				},
+			},
+		},
+	}
```

Comment:
@achilleas-k @thozza I dont see its getting added to the manifest, Not sure what I am missing.

Reply:
I see it added to the edge commit but not the edge container.  You did add it to the edge commit above (Line 27), so add it the same way to the edge container (Line 51) and it will generate the same manifests as before.